semanticsearch/README.md

```diff
@@ -0,0 +1,440 @@
+# Semantic Search Workload
+
+This workload is to benchmark performance of search of Semantic Search queries of OpenSearch. 
+
+## Datasets
+
+This workload is based on a [daily weather measurement from NOAA](ftp://ftp.ncdc.noaa.gov/pub/data/ghcn/daily/by_year/).
+
+To recreate the document corpus:
```

Comment:
If this needs to be performed each time the user sets up a load generation host, it might be better to have these steps included in a helper script. That way the user only needs to invoke the script and it'll automatically pull in the corpora file, shuffle docs, etc. 

Can the randomly shuffled corpora live in a cloudfront repository? If so, that would remove the need for the user to perform these steps.

Reply:
Do we even need this? I see corpora is added in workload.json.

Reply:
Correct, corpora is added in a workload.json. I copied this from original `noaa` workload. Will update the README and remove that block